prepare-build.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
-
-set -e
-
-cd build
-mv static resources
-mv favicon.png resources/
```

Comment:
I moved the favicon to the `static` folder initially, so no transformations are required after build